django/contrib/admin/utils.py

```diff
@@ -395,7 +395,7 @@ def display_for_field(value, field, empty_value_display):
     # before the general null test.
     elif isinstance(field, (models.BooleanField, models.NullBooleanField)):
         return _boolean_icon(value)
-    elif value is None:
+    elif value is None or isinstance(value, str) and value == '':
```

Comment:
I do't think these changes are needed, since the empty value can be handled by display_for_value function.

Reply:
Okay, I see it now, you're right